libs/parallelism/algorithms/tests/unit/algorithms/replace_copy.cpp

```diff
@@ -19,6 +19,36 @@
 #include "test_utils.hpp"
 
 ////////////////////////////////////////////////////////////////////////////
+template <typename IteratorTag>
+void test_replace_copy(IteratorTag)
```

Comment:
One small thing I noticed, you forget to add `test_replace_copy(IteratorTag());` so this test is never called